File: 12250.c
#include<stdio.h>
#include<stdlib.h>
#include<string.h>

int main(){
	int cont=0;
	char tmp[15];
	
	while(scanf("%s", tmp)) {
		if(strcmp(tmp,"#")==0)return 0;
		
		if(strcmp(tmp,"HELLO")==0){
			printf("Case %d: ENGLISH\n",++cont);
		}
		else if(strcmp(tmp,"HOLA")==0){
			printf("Case %d: SPANISH\n",++cont);
		}
		else if(strcmp(tmp,"HALLO")==0){
			printf("Case %d: GERMAN\n",++cont);
		}
		else if(strcmp(tmp,"BONJOUR")==0){
			printf("Case %d: FRENCH\n",++cont);
		}
		else if(strcmp(tmp,"CIAO")==0){
			printf("Case %d: ITALIAN\n",++cont);
		}
		else if(strcmp(tmp,"ZDRAVSTVUJTE")==0){
			printf("Case %d: RUSSIAN\n",++cont);
		}
		else
			printf("Case %d: UNKNOWN\n",++cont);
	}
}

File: 12250.py
from sys import stdin
def main():
	i = 0;
	while(True):
		i += 1
		palabra=str(input())
		if(palabra == "#"):
			break;
		elif(palabra == "HELLO"):
			print("Case "+str(i)+":"+" "+"ENGLISH")
		elif(palabra == "HOLA"):
			print("Case "+str(i)+":"+" "+"SPANISH")
		elif(palabra == "HALLO"):
			print("Case "+str(i)+":"+" "+"GERMAN")
		elif(palabra == "BONJOUR"):
			print("Case "+str(i)+":"+" "+"FRENCH")
		elif(palabra == "CIAO"):
			print("Case "+str(i)+":"+" "+"ITALIAN")
		elif(palabra == "ZDRAVSTVUJTE"):
			print("Case "+str(i)+":"+" "+"RUSSIAN")
		else:
			print("Case "+str(i)+":"+" "+"UNKNOWN")
main()